lighthouse-core/audits/first-interactive.js

```diff
@@ -22,8 +22,9 @@ class FirstInteractiveMetric extends Audit {
       category: 'Performance',
       name: 'first-interactive',
       description: 'First Interactive (beta)',
-      helpText: 'The first point at which necessary scripts of the page have loaded ' +
-          'and the CPU is idle enough to handle most user input.',
+      helpText: 'First Interactive marks the time at which the ' +
+          'page is interactive enough to handle most user input. ' +
```

Comment:
OK, but I'm gonna stick with `page` rather than website, so that we're not giving the impression that LH is somehow auditing other pages, as well